tests/FixtureBuilder/ExpressionLanguage/TokenTypeTest.php

```diff
@@ -48,7 +48,7 @@ public function testStaticValues(): void
 
         $reflProp = $reflClass->getProperty('values');
         $reflProp->setAccessible(true);
-        $values = $reflProp->getValue(TokenType::class);
+        $values = $reflProp->getValue((object) TokenType::class);
```

Comment:
??

Reply:
If run in PHP 8
```
1) Nelmio\Alice\FixtureBuilder\ExpressionLanguage\TokenTypeTest::testStaticValues
TypeError: ReflectionProperty::getValue(): Argument #1 ($object) must be of type ?object, string given

/home/runner/work/alice/alice/tests/FixtureBuilder/ExpressionLanguage/TokenTypeTest.php:50
```

Reply:
Maybe the reflection API changed for a static value, but the change does not look correct, `(object) TokenType::class === stdClass{ #scalar: TokenType::class }` which is most likely not what we want